app/router.js

```diff
@@ -7,7 +7,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+
+  this.route('reminders', function(){
+    this.route('reminder', { path: '/:id' });
```

Comment:
Would you mind changing :id to :reminder_id -> Ember follows the convention of :model-name_id for two reasons. The first reason is that Routes know how to fetch the right model by default, if you follow the convention. The second is that params is an object, and can only have one value associated with a key. 
https://guides.emberjs.com/v2.11.0/routing/defining-your-routes/